### ISSUE TITLE
Fix path problem for windows

### ISSUE THREAD

**Issue by EragonJ** [EragonJ/Kaku#287]
As title, after introducing `youtube-api`, it makes the hierarchy too complicated and when `unzip` Kaku on Windows, this will make the path exceed the MAX_LENGTH and will break the application.

**Comment by EragonJ** [comment]
Fuck, windows sucks !!!!

**Comment by Draxler** [comment]
Noooo ...

@EragonJ 

you need to unziped (while) on desktop. After that you could move it where ever you want.

**Comment by EragonJ** [comment]
I think that is quite the same because it hits the native limitation. To be honest, there is no good way to fix this because normal users can rename their folders to a REALLY_LONG_NAME and no matter how tried, this problem would happen sometimes somewhere, but this is an edge case and we can just ignore it.

Debugged from the log I got, there is only one 3rd party library which hits the limitation and if I can find a better alternative, this should be fixed. (Well, to be honest, that's the best 3rd party I found ...) Never mind, i'll keep this issue open and maybe rewrite a new one library by my own to make it work.

again, windows sucks !!!!!
